### DISCUSSION TITLE
Change/override bottom navigation bar’s height?

### DISCUSSION THREAD

**Discussion by inari-codes** [beercss/beercss#312] [Q&A] [answered]
I’m just making this design for fun, not sure if i can make something like Instagram with that 😝

Anyways I want to ask is it possible to manually override or use size helpers for the bottom navigation? Because the default height is quite large and I want to change it

The repo is here if you want to look at it: https://github.com/ItsKruistz/My-Site

![3F6E651D-CFDE-4BDD-B435-1AEF8664F906](https://github.com/beercss/beercss/assets/85934291/fb836808-a58b-45a7-8a58-267a5f419d5b)

**Comment by leonardorafael** [comment, marked as answer]
Hi @ItsKruistz ,

The default specs from MD3 says:

- Top app bar has 64dp
- Bottom app bar has 80dp

You can override the bottom app bar `<nav class="bottom">` to have `nav.bottom { block-size: 4rem; }`.
Or we can create/develop `small`, `medium` and `large` helpers to use as `<nav class="bottom small">` too.
I believe that, in theory, is possible to do a MD3 version of Instagram, Twitter, Facebook, using Beer CSS (without add any custom CSS).

Hope this helps 🍻
